packages/orbit-components/transforms/__tests__/Modal-size.test.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+const { defineTest } = require("jscodeshift/dist/testUtils");
+
+defineTest(__dirname, "Modal-size");
```

Comment:
It's best to test codemods with both Flow and TypeScript parser just to be sure that there is no funny business. You can see how to do this in `Stack-spacing` tests.